src/find.ts

```
@@ -0,0 +1,93 @@
+
+  // Remove fields that we added to the query (such as paginatedField and _id) that the user didn't ask for.
+  if (removePaginatedFieldInResponse && params.paginatedField) {
+    response.results = _.map(response.results, (result) => _.omit(result, params.paginatedField));
+  }
+
+  return response;
+}
+
+function hasFindAsCursor(
+  collection: unknown
+): collection is Collection<Document> & { findAsCursor: Function } {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)